lib/Doctrine/ODM/PHPCR/Mapping/Annotations/Document.php

```diff
@@ -66,4 +66,9 @@ class Document
      * @var boolean
      */
     public $uniqueNodeType;
+
+    /**
+     * @var array
```

Comment:
`array|null`.

The other option is to have this as a separate annotation, `@RestrictChidren({"Foo\ChildClass", restrict="whitelist")`. Which would also help us here as restrict woule be an `enum`: `none, whitelist, blacklist` or similar.